express-post-json/server.ts

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+
+type Student = {
+  id?: number;
```

Comment:
This `?` is an interesting choice. I definitely understand why you did it, because in the post handler the body is a Person without an id.

However, it's usually best to have a strongly typed object that matches what you will have in your data store, which is required to have an `id`.

The way I've done this is to use composition (though this is not required for this exercise since you don't actually declare the type in your post handler):
```
type UnsavedStudent = {
  some_fields: whatever;
}
type Student = UnsavedStudent & { id: number };
```

This way, `Student` is what is stored in my data store and is required to have an id. `UnsavedStudent` is what the client sends in the post request.